lib/matplotlib/backends/backend_qt4agg.py

```diff
@@ -54,13 +54,7 @@ def new_figure_manager_given_figure(num, figure):
     return FigureManagerQT(canvas, num)
 
 
-class FigureCanvasQTAggBase(_FigureCanvasQTAggBase):
-    def __init__(self, figure):
-        self._agg_draw_pending = False
-
-
-class FigureCanvasQTAgg(FigureCanvasQTAggBase,
-                        FigureCanvasQT, FigureCanvasAgg):
+class FigureCanvasQTAgg(FigureCanvasQTAggBase, FigureCanvasQT):
```

Comment:
I think having the base class come in as the `_` version of this is clearer (I had to think a bit for why this should work).